test/demo/utils/ofPuppeteer.ts

```diff
@@ -0,0 +1,117 @@
+import {Fin, Identity} from 'openfin/_v2/main';
+import * as puppeteer from 'puppeteer';
+import {Browser, Page, Target} from 'puppeteer';
+
+const fetch = require('node-fetch');
+
+// This gets mounted by jest as part of our setup
+declare const global: NodeJS.Global&{__BROWSER__: puppeteer.Browser};
+
+export type TestWindowContext = Window&{
+  fin: Fin;
+  OpenfinFDC3: typeof import('../../../src/client/main')
+};
+
+export class OFPuppeteerBrowser {
```

Comment:
Have only skimmed this so far, but if we're going to be wrapping Puppeteer in a util, can we make it more "OpenFin-y"?

As in, rather than using puppeteer's page and browser concepts, name things around "windows" and "apps"? Looking at the current util's, it's not clear what a page refers to exactly - a window?..

Reply:
Part of the issue with that is that there isn't really a good correlation between the two that I've been able to figure out yet. We should discuss offline and maybe chat with someone from runtime about what the mappings are.